app/models/jwt_secret.rb

```diff
@@ -0,0 +1,25 @@
+class JwtSecret < ActiveRecord::Base
```

Comment:
whats the difference between user auth tokens and jwt? should we keep both?

Reply:
The `JWTSecret` is a token used to encryt/decrypt the JWT token. I think it's a good idea to keep this separate. `PersonalAccessToken` can be used like a password. I think, they aren't really related.